src/lib.rs

```diff
@@ -43,7 +43,8 @@ pub fn run_compiled(book: &hvmc::ast::Book, mem_size: usize) -> (Net, RunStats)
   let start_time = Instant::now();
   root.normal(&runtime_book);
   let elapsed = start_time.elapsed().as_secs_f64();
-  let rewrites = Rewrites { anni: root.anni, comm: root.comm, eras: root.eras, dref: root.dref, oper: root.oper };
+  let rewrites =
+    Rewrites { anni: root.anni, comm: root.comm, eras: root.eras, dref: root.dref, oper: root.oper };
   let net = net_from_runtime(&root);
   //eprintln!("{}", show_net(&net));
```

Comment:
Remove this commented line

Reply:
This is some debug that I forgot and taelin commented out instead of removing